akka-docs/rst/java/http/http-model.rst

```diff
@@ -268,3 +268,9 @@ provided to parse (or render to) Strings or byte arrays.
   and can override them if needed. This is useful, since both ``client`` and ``host-connection-pool`` APIs,
   such as the Client API ``Http.get(sys).outgoingConnection`` or the Host Connection Pool APIs ``Http.get(sys).singleRequest``
   or ``Http.get(sys).superPool``, usually need the same settings, however the ``server`` most likely has a very different set of settings.
+
+URI
```

Comment:
I'd suggest:

```
The URI model
----------------- // align length
Akka HTTP offers it's own specialised URI model class which is tuned for both performance and idiomatic usage within other types of the HTTP model. For example, an HTTPRequest's target URI is parsed into this type, where all character escaping and other URI specific semantics are applied.

Obtaining the Raw Request URI
^^^^^^^^^^^^^^^^^ // subsection, align length of underling
Sometimes it may be needed to obtain the "raw" value of an incoming URI, without applying any escaping or parsing to it. While this use-case is rare, it comes up every once in a while. It is possible to obtain the "raw" request URI in Akka HTTP Server side by turning on the `....` flag. [.... and here the rest of your text"

```